compiler/src/dotty/tools/dotc/interactive/Interactive.scala

```diff
@@ -219,7 +219,7 @@ object Interactive {
       case _  => getScopeCompletions(ctx)
     }
     interactiv.println(i"completion with pos = $pos, prefix = $prefix, termOnly = $termOnly, typeOnly = $typeOnly = ${completions.toList}%, %")
-    (completionPos, completions.toList)
+    (completionPos, completions.toList.sortBy(_.name.toString))
```

Comment:
@smarter We have an ordering for `Name` that differs from the default `String` ordering. I am wondering if we should use it here? The Name ordering will not group type names and term names together

Reply:
I think they should be grouped together, but we should also not display type completions if we're not in a type context

Reply:
I think they should be grouped together, but we should also not display type completions if we're not in a type context

Reply:
Sure but that's probably out of the scope of this PR